src/networkd.c

```diff
@@ -1091,6 +1091,14 @@ append_wpa_auth_conf(GString* s, const NetplanAuthenticationSettings* auth, cons
             g_string_append(s, "  eap=TTLS\n");
             break;
 
+        case NETPLAN_AUTH_EAP_LEAP:
+            g_string_append(s, "  eap=LEAP\n");
+            break;
+
+        case NETPLAN_AUTH_EAP_PWD:
+            g_string_append(s, "  eap=PWD\n");
```

Comment:
question: could you please link some official reference for those wpa_supplicant settings? I couldn't find anything when looking it up quickly.

Reply:
Yeah this thing is not documented! I had to look it up in the hostapd/wpa_supplicant code to be sure it existed.

Example in test scripts:
https://w1.fi/cgit/hostap/tree/tests/hwsim/test_eap_proto.py#n7699

There is also a brief note about it here https://w1.fi/cgit/hostap/plain/hostapd/hostapd.conf